tests/__snapshots__/index.test.js.snap

```diff
@@ -152,7 +152,6 @@ Array [
       "--proxy-tunnel",
       "--verbose",
       "--tunnel-name=my-tunnel",
-      "--no-proxy-caching",
```

Comment:
Is `--no-proxy-caching` basically a no-op now?

Reply:
it's been no-op for the last several years